backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/configuration/ClusterConfiguration.java

```diff
@@ -232,6 +241,7 @@ public static Builder builder() {
     public static ClusterConfiguration from(Configuration configuration) {
         return ClusterConfiguration.builder()
             .hosts(listCassandraServers(configuration))
+            .createKeyspace(configuration.getBoolean(CASSANDRA_CREATE_KEYSPACE, true))
```

Comment:
I would say false by default

Reply:
& duplicate default value suppling detected.

Reply:
defaulting to false will break our docker-compose, no?

Reply:
We can always explicitly configure this to true in docker packaging